.github/workflows/wheels-ci.yml

```diff
@@ -23,8 +23,8 @@ jobs:
       commit: ${{ github.sha }}
     strategy:
       matrix:
-        os: ['macos-13', 'ubuntu-22.04']
-        python_version: ['3.9', '3.10', '3.11', '3.12', '3.13']
+        os: ['macos-13', 'ubuntu-24.04']
+        python_version: ['3.9', '3.14']
```

Comment:
Note that I removed the other versions because it was taking up a lot of CI time, I think just testing oldest and latest does the trick.